src/content/en/2019/fonts.md

```diff
@@ -0,0 +1,118 @@
+
+**06.44: font-display values**
+
+As an easy way to show fallback text while a web font is loading, `font-display: swap` reigns supreme and is the most common descriptor value. I would have expected `optional` (only render if cached) to have a bit more usage here as a few prominent developer evangelists lobbied for it a bit—but no dice.
+
+**06.09a: Web font requests per page**
+
+The median web page makes 4 web font requests. Even the 10th percentile requests one. The ubiquity of Web fonts is complete. It does seem interesting to me that web font requests seem to be pretty steady across desktop and mobile. I’m glad to see the recommendation to hide `@font-face` blocks inside of a `@media` query didn’t catch on (don’t do this).
+
+**06.34: Font requests per page**
+
+Why are there more requests for fonts made on mobile devices? Is it related to fewer typefaces available on mobile devices, which in turn means fewer `local()` hits in Google Fonts CSS? Actually we might be on to something here.
+
+**06.09b: Web font families**
+
+- [ ]  *Zach’s note: Is this families per page? What is `httparchive.almanac.parsed_css`?*
+
+These results were a little surprising. I didn’t expect this to line up so well with the number of web font requests per page. Namely I didn’t expect so many unique font families! To me this suggests a few things might be happening here:
+
+- Authors are mapping web fonts that share the same typeface to unique font-family names. My intuition seems to think this is unlikely.
+- Authors are only using one web font from each font-family (this will be more common as variable font usage increases).
+- I’ve seen some foundries (typography.com) use multiple font-family names as a copy-prevention mechanism for web fonts but I don’t think this is very common.
+
+**06.10: Italics**
+
+Only 29% of pages declare at least one italic font. This suggests a lot of faux-italic across the web, as web browsers will synthesize a fake bold version if one is not available for use. That means using elements with italic user agent styling (`em`, `i`, et al) will render faux-italic.
+
+Faux bold may be even larger of a problem, as about 53–55% of pages have a font-face with a weight above 400, and (arguably) bold usage is more common than italic usage.
+
+(Counterargument: a `@font-face` needn’t be declared bold or italic if the underlying glyphs are from a bold or italic variant of the typeface)
+
+ **06.13: CSS Font Loading API**
+
+This is the first metric that elicited an audible “oof” from me. The CSS Font Loading API is currently the only way to group web font repaints. Arguably unnecessary if you’re only using one font file per typeface, which per the implication in in `06.09b` may be common practice!
```

Comment:
Add a link to https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/webfont-optimization#the_font_loading_api